### PR TITLE
OSOE-859: Add option to verify-dotnet-consolidation to ignore projects matching a given pattern in Lombiq.GitHub.Actions

### PR DESCRIPTION
**Pull request by sarahelsaig**
[OSOE-859](https://lombiq.atlassian.net/browse/OSOE-859)

[OSOE-859]: https://lombiq.atlassian.net/browse/OSOE-859?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Lombiq.OSOCE.Web/Lombiq.OSOCE.Web.csproj [hunk 1]
```diff
@@ -58,10 +58,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="OrchardCore.Admin.Abstractions" Version="1.8.2" />
-    <PackageReference Include="OrchardCore.Application.Cms.Targets" Version="1.8.2" />
-    <PackageReference Include="OrchardCore.Logging.NLog" Version="1.8.2" />
-    <PackageReference Include="SixLabors.ImageSharp.Web" Version="3.1.0" />
+    <PackageReference Include="OrchardCore" Version="1.8.3" />
```
Comment (Piedone) [inline src/Lombiq.OSOCE.Web/Lombiq.OSOCE.Web.csproj]: Why is this needed?

**Review by Piedone — CHANGES_REQUESTED** [review]

File: src/Lombiq.OSOCE.Web/Lombiq.OSOCE.Web.csproj (hunk 1 above)
Comment (sarahelsaig) [reply src/Lombiq.OSOCE.Web/Lombiq.OSOCE.Web.csproj]: I was mistaken, it's not needed.

**Review by Piedone — APPROVED** [review]
